server/v2/api/grpc/server.go

```diff
@@ -35,10 +38,10 @@ type GRPCService interface {
 
 // New returns a correctly configured and initialized gRPC server.
 // Note, the caller is responsible for starting the server.
-func New(logger log.Logger, v *viper.Viper, interfaceRegistry appmanager.InterfaceRegistry, app GRPCService) (GRPCServer, error) {
+func New(logger log.Logger, v *viper.Viper, interfaceRegistry appmanager.InterfaceRegistry) (GRPCServer, error) {
 	cfg := DefaultConfig()
 	if v != nil {
-		if err := v.Sub(serverName).Unmarshal(&cfg); err != nil {
```

Comment:
I think the sub is still needed. Why is it not? The viper you get is from the all config.

Reply:
revert to use viper.Sub()